src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -333,4 +333,23 @@ public static void emptyTempTargetSpannerNewDBAndTable() {
     spannerTargetDatabase = StringUtils.EMPTY;
     spannerTargetTable = StringUtils.EMPTY;
   }
+
+  /**
+   * Create GCS bucket with csv file containing below data types:
+   * int, float, string, double, long, timestamp, date, boolean,bytes.
+   * */
+  @Before(order = 1, value = "@GCS_ALL_DATATYPES_FOR_SPANNER_SINK_TEST")
```

Comment:
Add newly added tags in @After hook 

Refer : https://github.com/data-integrations/google-cloud/blob/3d96300fdbfff74d66e3ac465f41e99aed5986f3/src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java#L85

Reply:
Remove @Before(order = 1, value = "@GCS_ALL_DATATYPES_FOR_SPANNER_SINK_TEST")